src/base/emu-i386/simx86/interp.c

```diff
@@ -2289,7 +2255,7 @@ intop3b:		{ int op = ArOpsFR[D_MO(opc)];
 				Gen(S_DI, mode);
 				break;
 			case Ofs_SP:	/*4*/	 // JMP near indirect
-				PC = JumpGen(PC, mode, 0x40,
+				PC = JumpGen(PC, mode, JMPi,
```

Comment:
But not opc? Why?

Reply:
opc is not sufficient here, as it's `0xff` (GRP2wrm) but were dealing with a 2 byte opcode of the form "ff /4" where "4" denotes the value of bits 3 - 5, so ff20 - ff27, ff60 - ff67, ffa0 - ffa7 and ffe0 - ffe7 in a disassembler. I defined JMPI as ff04 but could also use ff20 as some kind of "opcode base value", in that case one could use `(opc << 8) | (opc2 & 0x38)` ... would you like that?